### PR TITLE
chat history and insights

### PR DESCRIPTION
**Pull request by molt3nl3ad**
Chat history working and also created routes for separating insights

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ services:
       - MB_DB_PASS=${MB_DB_PASS}
       - REACT_APP_SEMANTIC=${REACT_APP_SEMANTIC}
       - REACT_APP_WEBSOCKET_SERVER=${REACT_APP_WEBSOCKET_SERVER}
-      - REACT_APP_SEMANTIC=${REACT_APP_SEMANTIC}
+      - REACT_APP_SERVER=${REACT_APP_SERVER}
```
Comment (jordigiam) [inline docker-compose.yml:20]: why this change?

**Review by jordigiam — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (molt3nl3ad) [reply docker-compose.yml:20]: because it was set twice

**Review by jordigiam — APPROVED** [review]
